### ISSUE TITLE
About backbones

### ISSUE THREAD

**Issue by hariniyum** [xrli-U/MuSc#23]
Hi, 
I'm so glad that I found your great work.
I have a quick question.

Have you tried your method with a pre-trained vit-base8?
I was thinking that small patch sizes would be helpful to get sophisticated representations.
So, I was wondering if you've done experiments with these backbones.

Thank you for your work again.

**Comment by xrli-U** [comment]
> Hi, 
> I'm so glad that I found your great work.
> I have a quick question.
> 
> Have you tried your method with a pre-trained vit-base8?
> I was thinking that small patch sizes would be helpful to get sophisticated representations.
> So, I was wondering if you've done experiments with these backbones.
> 
> Thank you for your work again.

You're right, using smaller patches is more advantageous for segmentation, but for fair comparison with other methods, our default backbone doesn't use vit-base8.
